### ISSUE TITLE
Change change-warning on all services to warning, and update health checks with name, description, and severity keys

### ISSUE THREAD

**Issue by chrisguida** [Start9Labs/embassy-os#832]
**Describe the bug**
I can't get change warnings to display on the FE

**To Reproduce**
- Install bitcoind
- hit "config"
- change any of the pruning settings. all of them have change warnings defined in the spec

**Expected behavior**
A change warning should pop up when toggling the variant between disabled, automatic, and manual variants. In addition, the change warnings on altering the pruning sizes 

**Actual behavior**
No warning appears

**Desktop (please complete the following information):**
 - OS and version: Ubuntu 20.04
 - Browser and version: FF 94.0

**Additional context**
here's the relevant portion of the bitcoind config spec:
```
    pruning:
      type: union
      name: "Pruning Settings"
      description: |
        Blockchain Pruning Options
        Reduce the blockchain size on disk
      change-warning: |
        If you set pruning to Manual and your disk is smaller than the total size of the blockchain, you MUST have something running that prunes these blocks or you may overfill your disk!
        Disabling pruning will convert your node into a full archival node. This requires a resync of the entire blockchain, a process that may take several days. Make sure you have enough free disk space or you may fill up your disk.
      tag:
        id: mode
        name: Pruning Mode
        description: |
          - Disabled: Disable pruning
          - Automatic: Limit blockchain size on disk to a certain number of megabytes
          - Manual: Prune blockchain with the "pruneblockchain" RPC
        variant-names:
          disabled: Disabled
          automatic: Automatic
          manual: Manual
      variants:
        disabled: {}
        automatic:
          size:
            type: number
            nullable: false
            name: "Max Chain Size"
            description: "Limit of blockchain size on disk."
            change-warning: "Increasing this value will require re-syncing your node."
            default: 550
            range: "[550,1000000)"
            integral: true
            units: MiB
        manual:
          size:
            type: number
            nullable: false
            name: "Failsafe Chain Size"
            description: "Prune blockchain if size expands beyond this."
            default: 65536
            range: "[550,1000000)"
            integral: true
            units: MiB
      default: disabled
```

git commit is `613b05788ba929679f62a03d34b15802ee4a5c37`, kind of old at this point

**Comment by dr-bonez** [comment]
@chrisguida please confirm that the change warnings are coming back from the network tab. If so assign to @DrewAns otherwise assign to me

**Comment by chrisguida** [comment]
Here's what's coming back under the "pruning" key. Not seeing "change-warning" anywhere. Assigning @dr-bonez 

```
{
    "default": "disabled",
    "description": "Blockchain Pruning Options\nReduce the blockchain size on disk\n",
    "display-as": null,
    "name": "Pruning Settings",
    "tag": {
        "description": "- Disabled: Disable pruning\n- Automatic: Limit blockchain size on disk to a certain number of megabytes\n- Manual: Prune blockchain with the \"pruneblockchain\" RPC\n",
        "id": "mode",
        "name": "Pruning Mode",
        "variant-names": {
            "automatic": "Automatic",
            "disabled": "Disabled",
            "manual": "Manual"
        }
    },
    "type": "union",
    "unique-by": null,
    "variants": {
        "automatic": {
            "size": {
                "default": 550,
                "description": "Limit of blockchain size on disk.",
                "integral": true,
                "name": "Max Chain Size",
                "nullable": false,
                "range": "[550,1000000)",
                "type": "number",
                "units": "MiB"
            }
        },
        "disabled": {},
        "manual": {
            "size": {
                "default": 65536,
                "description": "Prune blockchain if size expands beyond this.",
                "integral": true,
                "name": "Failsafe Chain Size",
                "nullable": false,
                "range": "[550,1000000)",
                "type": "number",
                "units": "MiB"
            }
        }
    }
}

```

**Comment by chrisguida** [comment]
Also found another issue, the description under the `tag` key isn't rendering anywhere either: 

`"description": "- Disabled: Disable pruning\n- Automatic: Limit blockchain size on disk to a certain number of megabytes\n- Manual: Prune blockchain with the \"pruneblockchain\" RPC\n"`

**Comment by chrisguida** [comment]
> Also found another issue, the description under the `tag` key isn't rendering anywhere either:
> 
> `"description": "- Disabled: Disable pruning\n- Automatic: Limit blockchain size on disk to a certain number of megabytes\n- Manual: Prune blockchain with the \"pruneblockchain\" RPC\n"`
> 
> This one is a FE issue, unless this is no longer a valid way to display a description

Actually this one isn't showing on the FE anywhere on 02x either. Perhaps we should just remove it from the spec?

**Comment by dr-bonez** [comment]
Ok I have reviewed the specification. The field is no longer called `change-warning`, it is called `warning`. This is because the warning is shown always, not just when the field is changed. @chrisguida please change the field in your spec.

**Comment by chrisguida** [comment]
Cool, this needs to be updated on all packages then @elvece

**Comment by ProofOfKeags** [comment]
Closing as won't fix

**Comment by chrisguida** [comment]
@ProofOfKeags we still need to track the services work somewhere, reopening until that exists

**Comment by chrisguida** [comment]
Tracking here @elvece :

- [ ] synapse
- [ ] mastodon
- [ ] bitcoind
- [ ] ride-the-lightning
- [ ] thunderhub
- [ ] c-lightning
- [ ] vaultwarden
- [ ] filebrowser
- [ ] pages
- [ ] lnd
- [ ] btcpay
- [ ] photoview
- [ ] proxy
- [ ] spark
- [ ] bar
- [ ] cups
- [ ] sphinx

**Comment by chrisguida** [comment]
@dr-bonez do you have any thoughts on this, or should i create a new ticket?

https://github.com/Start9Labs/embassy-os/issues/832#issuecomment-978021827

**Comment by chrisguida** [comment]
The above checked services have this complete:

```
- change change-warning to warning
- change critical: true to severity: critical, and critical: false to severity: warning
- all health checks will now have a name: <name> field and a description: <desc> field
```

**Comment by chrisguida** [comment]
Renaming ticket to reflect all changes that need to happen to `manifest.yaml` and `config_spec.yaml`, since it's convenient to update both files at once.

**Comment by ProofOfKeags** [comment]
> Waiting for spec changes to be reflected in embassy-sdk pack before rebuilding s9pks
as of 4a143ea, you should be able to repack

**Comment by dr-bonez** [comment]
> @dr-bonez do you have any thoughts on this, or should i create a new ticket?
> 
> https://github.com/Start9Labs/embassy-os/issues/832#issuecomment-978021827

We should fix this on the FE

**Comment by chrisguida** [comment]
> > @dr-bonez do you have any thoughts on this, or should i create a new ticket?
> > [#832 (comment)](https://github.com/Start9Labs/embassy-os/issues/832#issuecomment-978021827)
> 
> We should fix this on the FE. Create a ticket for it.

Done

**Comment by chrisguida** [comment]
These services are now all updated and published to the beta 0.3 reg https://github.com/Start9Labs/embassy-os/issues/832#issuecomment-980141784
